roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -23,4 +23,5 @@
     name: https://fedorapeople.org/groups/katello/releases/yum/{{ foreman_repositories_katello_version }}/katello/el{{ ansible_distribution_major_version }}/x86_64/katello-repos-latest.rpm
     disable_gpg_check: true
     state: present
+  retries: 3
```

Comment:
it feels wrong to put in a workaround for our CI into the role itself, not that I'd have a better solution :(

Reply:
I was hoping you would, cause re-running the job 12 times yesterday felt bad and this felt just enough less bad to be worth it.

One solution is to move Katello repos off of fedorapeople :P